signal_analog/charts.py

```diff
@@ -436,6 +436,10 @@ def with_chart_legend_options(self, dimension, show_legend=False):
         self.chart_options.update({'onChartLegendOptions': opts})
         return self
 
+    def include_zero(self, boolean):
```

Comment:
can you make it `with_include_zero` or `is_include_zero` just to be consistent with other methods

Reply:
Ha! I think all the methods should be consistent and that's what we've been doing with all the latest changes. I am ok with modifying them. I'll let @dogonthehorizon confirm just to make sure there isn't any other reason for naming em that way